config-translate of Chinese.yml

Rainsry opened the pull request:
config-translate of Chinese.yml

 Could you please allow me to reprint your plugin to MCBBS ? (www.mcbbs.net) which is the largest Minecraft forum in China .You know not all of them are good at English, they can't read your post (Even with Google translator) they need a Chinese translator.People in the MCBBS will love your plugin .
I'll NEVER claim it as mine, I just wanna translate it and let more people know it. And I will mark out your (author’s) name and the download link from spigotmc.org
Thank you ! Hope for your reply!
Here is the translation of the config 

```
# StackMob 5.0 Configuration file - by antiPerson and contributors.
#
# Most options can be customised for specific entity types, for details about this, see the end of this file.
# Asterisk meaning:
# (*) This option cannot be overridden in the 'custom' section at the end of this file.
# (**) The following option requires the use of Paper (https://papermc.io) because the API needed is not in Spigot.
#
# If the comments have been removed, you can find a version with comments at https://github.com/Nathat23/StackMob-5/tree/master/src/main/resources

# StackMob 5.0配置文件-由antiPerson和贡献者们提供。
# 翻译汉化：Rainsry
# 大多数选项可以针对特定的生物类型进行自定义，有关详细信息，请参见此文件的末尾。
# 星号（*）的意义：
# （*）一个星号：一个*中的选项中的内容不能出现在此文件结尾的 “Custom” 中。
# （**）两个星号：两个*中的需要需要使用 Paper（https://papermc.io），因为所需的API不在spigot中
# 如果已删除注释，则可以在GitHub中找到含注释的版本 https://github.com/Nathat23/StackMob-5/tree/master/src/main/resources
stack:
  # The maximum size that a stack can have.
  # 一个堆叠后生物可以合并的最大数量。
  max-size: 30
  # How often entities should be checked for stacking (in ticks. 20 ticks = 1 second) (*)
  # 应该多久检查一次生物的堆叠情况（以ticks为单位。20个ticks = 1秒钟）（*）
  interval: 100
  # The area around an entity for which applicable stacks should be found.
  # Format: [(x cord),(y cord),(z cord)]
  # 对生物进行堆叠的检测范围是多少
  # 格式：merge-range: [x ,z ,y ] （xz为横向半径 y为纵向半径）
  merge-range: [10,10,10]
  # Only begin to stack entities when this amount of entities are nearby.
  # 仅在生物数量达到或超过所设定的值时才开始堆叠生物
  # 格式：amount: 数量
  threshold:
    enabled: true
    amount: 5

# Names of worlds where there should be no stacking
# 不进行生物堆叠的世界的名称
# 默认情况为黑名单，若想使用白名单，则将“worlds-blacklist-inbert: false ”设置为“true”
# 意为变为白名单
# 使用格式：worlds-blacklist: [世界名]
worlds-blacklist: []
worlds-blacklist-invert: false
# Types of entity which should not stack
# 不进行堆叠的生物种类
# 使用格式：types-blacklist: [生物名]
types-blacklist: [VILLAGER]
types-blacklist-invert: false
# If the entity's spawns reason is listed below, do not stack.
# 如果生物的生成原因是下方列表中所有的，则该生物将不会进行堆叠
reason-blacklist: []
reason-blacklist-invert: false

display-name:
  # The formatting of the name tag which is shown when the stack size is above the threshold.
  # Placeholders:
  # %type% - the entity's type
  # %size% - the size of the stack
  # 有堆叠生物时堆叠的主体显示的名称
  # 占位符：
  # %type% - 生物的类型
  # %size% - 堆叠的数量
  format: '&a%type% &d(%size%)'
  # Whether the display tag can only be seen when the player targets the entity.
  # Mode:
  # ALWAYS - The tag will always be visible as long as the entity is loaded.
  # HOVER - The tag will only be visible when the player hovers their crosshair on the entity.
  # NEARBY - The tag will only be visible when the player is in range of the entity. (more resource intensive, requires ProtocolLib on 1.15 servers)
  # 名称是否仅在玩家将该生物作为目标时才能看到。
  # 模式：
  # ALWAYS - 只要生物已加载，名称就始终可见。
  # HOVER  - 仅当玩家将十字准星悬停在生物上时，名称才可见。
  # NEARBY - 仅当玩家生物范围内时，名称才可见。 （此选项需要更多资源，且在1.15服务器上需要ProtocolLib）
  visibility: NEARBY
  # Options for when 'NEARBY' is used above. (*)
  # 上方设置使用“ NEARBY” 时的选项。（*）
  nearby:
    # The range from which the tag should be visible.
    # Format: [(x cord),(y cord),(z cord)]
    # 标签可见的范围。
    # 格式：[X , Z , Y ]
    range: [12,6,12]
    # How often (in ticks) the tag visibility status of an entity should be updated. (20 ticks = 1 second)
    # 生物的名称可见性状态应多久更新一次（以ticks为单位，20 ticks = 1 秒钟）
    interval: 20
  # Don't show the tag if the stack size of this entity if it is equal to or below the value specified.
  # 如果此生物的堆叠数量等于或小于指定的值，则不显示标签。
  threshold: 1

# Whether entity specific traits (eg. profession, colour) should be segregated (*)
# 是否应区分生物的特定特征（例如，职业，肤色）（*）
traits:
  sheep-color: true         # 是否保留可爱的羊咩咩的羊毛颜色
  sheep-sheared: true       # 是否保留可爱的被剪过羊毛的羊咩咩
  slime-size: true          # 是否保留可爱的史莱姆的尺寸大小
  horse-color: true         # 是否保留不同马的皮毛颜色
  llama-color: true         # 是否保留不同可爱的羊驼的颜色
  parrot-variant: true      # 是否保留不同鹦鹉的种类
  drowned-hand-item: true   # 是否掉落怪物（如僵尸）手中持有的物品
  age: true                 # 是否维持堆叠生物的不同年龄
  breed-mode: true          # 是否能进行繁殖
  love-mode: true           # 是否能进入恋爱/爱情模式（牛/猪/羊……）
  cat-type: true            # 是否保留猫的不同类型
  mooshroom-variant: true   # 是否保留不同蘑菇牛的种类
  fox-type: true            # 是否保留不同可爱的小狐狸的种类
  bee-nectar: true          # 堆叠后的蜜蜂是否能进行采蜜
  bee-stung: true           # 堆叠后的蜜蜂是否能蜇人
  zoglin-baby: true         # 猪宝宝是否能进行堆叠
  piglin-baby: true         # 疣猪兽的宝宝是否能进行堆叠
  leashed: true             # 堆叠后的生物是够能被栓绳操作

# Prevent stacked mobs from targeting players.
# Similar to no-ai, but allows for movement of entities.
# 是否防止堆叠后的怪物攻击玩家
# 类似于将怪物变成没有AI的生物，但是可以进行移动
# 下方的“enable:false”设置为“false”时，即为进行堆叠后的怪物会攻击玩家；设置为“true”时，即为进行堆叠后的怪物不会攻击玩家
disable-targeting:
  enabled: false
  # If the entity's type is listed below, do not disable targeting.
  # 如果某生物出现在下方的列表中，则所堆叠的怪物对玩家进行攻击
  # 格式：type-blacklist: [生物名]
  type-blacklist: []
  type-blacklist-invert: false
  # If the entity's spawns reason is listed below, do not disable targeting (**)
  # 如果生物的生成原因是下方列表中所有的，则该生物
  reason-blacklist: []
  reason-blacklist-invert: false

# What should be done when these entity actions occur. (*)
# 当这些生物动作发生时应该做些什么。(*)
events:
  # When an entity is feed its food.
  # 当一个生物被喂食后，将进行繁殖时
  breed:
    enabled: true
    # MULTIPLY - spawn as many baby entities possible providing there is enough food fed.
    # SPLIT - slice off an entity to that it can be bred normally.
    # 模式：
    # MULTIPLY - 如果有足够的食物喂养，则堆叠的生物尽可能多地进行繁殖。
    # SPLIT    - 将所堆叠的生物一个个“切割”开来（解除堆叠状态），成为正常的生物，然后进行繁殖
    mode: MULTIPLY
  # When an entity is dyed.
  # 当生物被染色时（比如羊咩咩被染色了）
  dye:
    enabled: true
    # MULTIPLY - dye as many entities possible providing there is enough dye.
    # SPLIT - slice off an entity to that it can be dyed normally.
    # 模式：
    # MULTIPLY - 如果有足够的染料，则会尽可能多的染色
    # SPLIT    - 将所堆叠的生物一个个“切割”开来（解除堆叠状态），然后进行染色
    mode: MULTIPLY
  # When an entity is sheared.
  # 当一个生物被剪刀进行操作时。（比如羊咩咩被剪羊毛）
  shear:
    enabled: true
    # MULTIPLY - shear as many entities possible providing the shears have enough durability.
    # SPLIT - slice off an entity to that it can be sheared normally.
    # 模式：
    # MOUTIPLY - 只要剪刀有足够的耐久。则会尽可能多地进行剪（羊毛）
    # SPLIT    - 将所堆叠的生物一个个“切割”开来（解除堆叠状态），然后进行剪（羊毛）处理
    mode: MULTIPLY
  multiply:
    # Chicken eggs and turtle scutes.
    # （可爱的小鸡生的）鸡蛋和（乌龟）鳞甲
    drops: true
    explosion: true
    slime-split: true
  divide:
    nametag: true
    tame: true
    # Only bees now
    # 这里仅仅是关于蜜蜂的配置
    enter-block: true
  remove-stack-data:
    # Should we remove the stack data of an entity that was unstacked as a result of nametag division/split?
    # Setting this to false will cause entities that were nametagged to potentially become stacked again.
    # 我们是否应该删除由于命名牌而未进行堆叠的生物的数据（是否堆叠由命名牌标记后的生物）
    # 将“nametag”设置为“false”时，则将导致被命名牌标记后的生物仍会被堆叠
    # 设置为“true”时，则被命名牌所标记后的生物不会被堆叠
    nametag: true

# How entities should die
# 堆叠后的生物的死亡方式
death:
  # Death options:
  # SINGLE - Only one entity dies.
  # ALL - Every entity dies.
  # STEP - A random amount dies.
  # STEP_DAMAGE - An amount that depends on the damage done dies.
  #
  # The priority 1 is highest and 4 is lowest.
  # If the killed entity is blacklisted, either by type or death reasons, by the highest priority method,
  # then it will attempt to use the next highest priority death method.
  # 死亡方式的几种不同选项：
  #
  # SINGLE - 在击杀所堆叠的生物时，每次击杀只会死亡一个生物
  # ALL    - 在击杀所堆叠的生物时，会一次性杀死堆叠生物中包含的全部生物
  # STEP   - 在击杀所堆叠的生物时，每次击杀所死亡的生物是随机的
  # STEP_DAMAGE - 在击杀所堆叠的生物时，每次击杀所死亡的生物取决于伤害大小
  #
  # 此配置中 1 为最高优先级， 4 为最低优先级
  # 请注意，此处的优先级（priority）是从 1 到 4 依次执行
  # 插件将使用最高优先级执行堆叠生物的死亡方式，
  # 如果生物的死亡原因/生物的类型处于黑名单中，将尝试使用下一个优先级的死亡方式，直到成功执行. 或者全部失败。
  SINGLE:
    priority: 4
    reason-blacklist: []
    reason-blacklist-invert: false
    type-blacklist: []
    type-blacklist-invert: false
  ALL:
    priority: 3
    reason-blacklist: []
    reason-blacklist-invert: false
    type-blacklist: []
    type-blacklist-invert: false
  STEP:
    priority: 2
    reason-blacklist: []
    reason-blacklist-invert: false
    type-blacklist: []
    type-blacklist-invert: false
    max-step: 5
    min-step: 1
  STEP_DAMAGE:
    priority: 1
    reason-blacklist: []
    reason-blacklist-invert: false
    type-blacklist: []
    type-blacklist-invert: false

# Multiply entity drops on entity death.
# 生物死亡时，生物掉落的物品是否乘以堆叠的数量
drops:
  enabled: true
  use-loot-tables: true
  # If each mob should only drop one of the items in the list.
  # 如果每个怪物死亡时只会掉落列表中的一项物品
  one-per-stack: []
  one-per-stack-invert: false
  # Items that should not be dropped.
  # 生物死亡时不会掉落的物品
  item-blacklist: []
  item-blacklist-invert: false
  # Death reasons that should not mean that drops are multiplied.
  # 生物死亡时不会掉落乘以数量时的死亡原因
  reason-blacklist: []
  reason-blacklist-invert: false
  # Types of entity for which drops should not be dropped for.
  # 死亡时不会掉落物品的生物种类
  type-blacklist: []
  type-blacklist-invert: false

# Multiply entity experience on entity death.
# 生物死亡时，生物掉落的经验是否乘以堆叠的数量
experience:
  enabled: true
  # The bounds that should be used for the random multiplier.
  # 对掉落经验时进行随机乘数的限制范围（最小范围/最大范围）
  multiplier-min: 0.5
  multiplier-max: 0.8
  # Types of entity for which exp should not be dropped for.
  # 死亡时不会掉落经验的生物种类
  type-blacklist: []
  type-blacklist-invert: false

# Multiply the player stats which can be seen by the player from the game menu.
# 乘以玩家可以从游戏菜单看到的玩家统计数据。
player-stats: true

# If specific entities should have a waiting time before their first stack.
# Designed for monster grinders etc.
# 是否设置特定生物进行堆叠前，有一段等待时间
# 用于刷怪塔等设施（防止生成的怪物在空中掉落时进行堆叠）
wait-to-stack:
  enabled: true
  # For (x) times the stack task fires after spawn, don't stack this entity.
  # 在生物生成（X）times 后，才会开始对该生物进行堆叠
  wait-time: 5
  # Entity types that this should work for.
  # 以下是适用于此设置（延迟堆叠）的生物
  types-whitelist:
    - ZOMBIE
    - SKELETON
    - PIG_ZOMBIE
    - CREEPER
    - ENDERMAN
  types-whitelist-invert: false
  # Spawn reasons that this should work for.
  # 由刷怪笼而生成的生物也会进行延迟堆叠
  reasons-whitelist:
    - SPAWNER
  reasons-whitelist-invert: false

# Enable/disable integration with other plugins. (*)
# 启用/禁用与其他插件的挂钩组合。（*）
hooks:
  # Allows the custom 'entity-stacking' flag to be used in worldguard regions.
  # 允许自定义 “生物堆叠” 适用于worldguard区域。
  worldguard: true
  mythicmobs:
    enabled: true
    blacklist: []
    blacklist-invert: false
  # Prevent citizens npcs from stacking.
  # 是否防止 Citizens NPC 进行堆叠
  citizens: true
  # Prevent stacked entities from giving mcmmo experience.
  # 是否防止堆叠的生物提供 MCMMO 经验
  mcmmo: true
  # Prevent stacked entities from giving job payments.
  # 是否防止堆叠后的生物提供“job payment” 
  jobs: true
  # Allows the use of ProtocolLib, which is used when the server version is not the native one of the plugin.
  # 是否允许使用 Protocol Lib
  protocollib: true

## Some options can be modified for specific entity types. An example is shown below. Remove comments to see this in action.
## Options and sections with a (*) in the comment preceding cannot be overridden.
#custom:
#  # The entity type
#  CREEPER:
#    # The option to override. This should be the same as it appears above.
#    stack:
#      max-size: 30
#  SKELETON:
#    # Entities can also inherit options from other entities, using the 'clone' option preceded by the entity type to clone.
#    clone: CREEPER
#    # The cloned custom options can still be overridden.
#    stack:
#      max-size: 5
#
# 您可以使用下方的配置，针对某些特定生物修改某些选项。示例如下所示（以一只可爱的苦力怕做示例）
# 在上方所有阐述的配置中，注释中有一个*号的内容，不能在下方出现！
#custom:
#  # The entity type 
#  CREEPER:
#    # 此处的自定义设置，能覆盖上方所进行的全局设置。
#    stack:
#      max-size: 30
#  SKELETON:
#    # 生物也可以从其他生物的自定义设置中继承设置，通过“clone”
#    # “clone”用于复制其他生物的自定义设置，并覆盖之前所定义的值
#    clone: CREEPER
#    # “clone” 中的设置，仍然可以在下方"stack"中进行二次重写，并覆盖之前的定义
#    stack:
#      max-size: 5
# 设置好之后，你就会发现，苦力怕的最大堆叠数量变成了5个
# 好啦，翻译到这里就结束了~如果某处翻译有误，请第一时间在MCBBS通知我
# 我会给予1人气20金粒的奖励~
```

Aurelien30000 commented:
Wow thanks for that, but currently, there is no way to really include it on the plugin itself, maybe on StackMob's spigot page & a link in the default config!

ploppyperson committed:
Add null check

ploppyperson committed:
Use HashMap rather than HashSet to store loaded stacked entities.

ploppyperson committed:
Add ClearLagg support.

ploppyperson committed:
Add to README.md

ploppyperson committed:
Increment version

ploppyperson committed:
Forgot to register ClearLagg hook...

ploppyperson committed:
April

ploppyperson committed:
Increment version.

ploppyperson committed:
Fix entities not being registered on Paper servers.

ploppyperson committed:
Increment version.

ploppyperson committed:
Use CodeMC nms repository

ploppyperson committed:
It is no longer the first day of April.

ploppyperson committed:
Update bStats

ploppyperson committed:
Revert "Update bStats"

This reverts commit a4f63409

ploppyperson committed:
CodeMC Nexus deployment

ploppyperson committed:
Add details to README

ploppyperson committed:
Optimise death drops generation.

ploppyperson committed:
Misc changes

ploppyperson committed:
Only drop items from block above when shearing

Aurelien30000 committed:
Add divide option on bucket fill (#210)

* Add config option for divide on bucket-fill

* Add BucketListener

Aurelien30000 committed:
Add VillagerProfession trait (#211)

* Add villager-profession trait to config

* Register villager trait

* Create VillagerProfession.java

ploppyperson committed:
Add MyPet hook

ploppyperson committed:
Store config gets in variables

ploppyperson committed:
Increment version

ploppyperson committed:
Use base value for KillStepDamage unless a potion effect is active.

ploppyperson committed:
Fix compilation error.

ploppyperson committed:
Fix spelling of ClearLag

ploppyperson committed:
Add missing soft depends

ploppyperson committed:
Remove player requirement for /sm remove

ploppyperson committed:
Move turtle trait registration

Commit:
Fix trait version check.

ploppyperson committed:
Add option to disable stacking on spawn

ploppyperson committed:
Set stack size when spawning entity.

ploppyperson committed:
Increment version

ploppyperson committed:
Increment version

ploppyperson committed:
Fix error when durability of shears in dispenser is 1.

ploppyperson committed:
New health value is too high - Maybe fix it this time?

ploppyperson committed:
cleanup

ploppyperson committed:
Increment version

ploppyperson committed:
5.6 (#220)

Minecraft 1.17 compatibility.

Co-authored-by: Aurélien <43724816+Aurelien30000@users.noreply.github.com>

Aurelien30000 committed:
Minor fixes

Improve package checking using non-deprecated method.
Fix paper check always failing due to package not loaded by ClassLoader.

ploppyperson committed:
Use non depreciated method

ploppyperson committed:
Merge branch 'Aurelien30000-patch-1'

ploppyperson committed:
Add temporary fix for entity loading issues in 1.17 builds (#223)

ploppyperson committed:
Increment version

ploppyperson committed:
Use 1.17 Paper repo, remove temporary reflection

ploppyperson committed:
Check if entity is being removed during merge task

ploppyperson committed:
Increment version

ploppyperson committed:
Increase interval of spigot compatibility task.

ploppyperson committed:
Minecraft 1.17.1 and revert to Java 11.

ploppyperson committed:
Fix TagTask bugs

ploppyperson committed:
Fix TagTask not running on 1.16 servers

ploppyperson committed:
Fix BucketListener, support Axolotl (fixes #227)

ploppyperson committed:
Add more debug info (#230)

ploppyperson committed:
Check if stack threshold is disabled.

ploppyperson committed:
5.7 (#231)

* Add TagTask raytracing.

* Add nag about console logging on named mob death

* Return boolean rather than ConfigList where possible

* Add EntityGrouping enum

* Implement entity type groups

* Move paper check for spawn reason to config method

* Add stacking tool INFO mode

* Change config defaults

* Might help with #230

* Add paper check

* Increment version

ploppyperson committed:
5.7.1 (#233)

* Add missing toString, make ConfigList#contains have String parameter

* Remove TemporaryCompat

* Increment version

ploppyperson committed:
Fix config bug

ploppyperson committed:
Add null check

ploppyperson committed:
Add slice mode for entity explosion (#234)

ploppyperson committed:
Copy stack size for all entity transformation (fixes #236)

ploppyperson committed:
Update for 1.18 and increment version

ploppyperson committed:
Fix paper detection (fixes #238)

ploppyperson committed:
Increment version

ploppyperson committed:
Prevent transformed entities from becoming a stack entity, when the original entity was not stacked (fixes #242)

ploppyperson committed:
Check if world has changed before checking if entity has moved

ploppyperson committed:
Only print error message once.

ploppyperson committed:
Update for Minecraft 1.18.2

ploppyperson committed:
Increment version

ploppyperson committed:
Update for MythicMobs 5

ploppyperson committed:
Update for MythicMobs 5 (pt2)

ploppyperson committed:
Fix stack data not removed when entity is spawned by transformation

ploppyperson committed:
Increment version

cpaca committed:
Allow stacked villagers to always drop items. (#249)

Note: The only time villagers drop items is when a player with Hero of the Village appears.

ploppyperson committed:
Fix compile error

ploppyperson committed:
Update for Spigot 1.19

ploppyperson committed:
Support breeding frogs

ploppyperson committed:
Add frog variant trait

ploppyperson committed:
Fix stack task ending early.
This has been incorrect for two years.

ploppyperson committed:
Increment version

ploppyperson committed:
Update dependencies

ploppyperson committed:
5.8 (#253)

* Update version

* Tag visibility NEARBY improvements:

- Only send packets to player if the visibility state changes
- Use fake armor stand to display stack sizes.

* Improve command handling

- Use TreeMap rather than list, this means the sorting is automatic.
- Rewrite tab completion
- Give command feedback when sub command arguments are incorrect

* Add ability to spawn multiple stacks

* Config handling changes.

We now determine whether a section has been overridden when the plugin loads, rather than each time the config is accessed. The results are stored in an EntityConfig for each EntityType.

* Store EntityConfig in StackEntity

* Copy comments when updating config

* Remove warning for console messages


* Commands additions


* Change lore not item name when tool mode is changed

ploppyperson committed:
Use bukkit library loader rather than shading (fixes #254)

ploppyperson committed:
Fix spigot support

ploppyperson committed:
Use apache commons 3

ploppyperson committed:
Update maven shade plugin

ploppyperson committed:
Use legacy component serializer rather than whatever we were doing before

ploppyperson committed:
Use concurrent hash map

ploppyperson committed:
Add adventure dependency

ploppyperson committed:
Use more adventure

ploppyperson committed:
Increment version

ploppyperson committed:
Fix pom

ploppyperson committed:
Attempt to fix #256

ploppyperson committed:
Fix config path for slime split

ploppyperson committed:
Check world when ray tracing

ploppyperson committed:
EntityConfig add null checks

ploppyperson committed:
Fix swapped armor stand adjustment

ploppyperson committed:
Compile against 1.19.1

ploppyperson committed:
another attempt at fixing #256

ploppyperson committed:
Fix forcestack stacking blacklisted types

ploppyperson committed:
Fix use of generic config when an entity is present

ploppyperson committed:
Add allay owner trait (#257)

ploppyperson committed:
Type genericfy traits

ploppyperson committed:
Add abstract class to store the class object of the type parameter

ploppyperson committed:
Revert "Add abstract class to store the class object of the type parameter"

This reverts commit c3c343060d91840908fe31b81d6484cec0422092.

ploppyperson committed:
Use map to store entity traits

ploppyperson committed:
Fix allay not having version check

ploppyperson committed:
Add null checks

ploppyperson committed:
Fix incorrect config path

ploppyperson committed:
Use EnumMap

ploppyperson committed:
Increment version

ploppyperson committed:
Add apache commons to plugin.yml libraries

ploppyperson committed:
Add null check (fixes #258)

ploppyperson committed:
Use ternary operator

ploppyperson committed:
Don't tab complete if sub command argument is incorrect

ploppyperson committed:
Check magnitude of vector

ploppyperson committed:
Move ray trace function

ploppyperson committed:
Add ray tracing to check if stacks can see each other

ploppyperson committed:
Update dependencies

ploppyperson committed:
Don't give stack data to entities that belong to hooks

ploppyperson committed:
Change nametag handling

ploppyperson committed:
Fix update downloader

ploppyperson committed:
Fix setting of config comments

ploppyperson committed:
Update README.md

ploppyperson committed:
Increment version

ploppyperson committed:
Revert "Increment version"

This reverts commit 76f07b9d0ff6e73913d9b54ddfef84e845cbeebd.

ploppyperson committed:
Fix traits not being assigned for every applicable entity type

ploppyperson committed:
Increment version

ploppyperson committed:
Don't remove nametags

ploppyperson committed:
Try shading again

ploppyperson committed:
Fix 1.16.5 support

ploppyperson committed:
Fix casting error (#260)

ploppyperson committed:
Remove cast (#260)

ploppyperson committed:
Fix command validation when not enough required args provided

ploppyperson committed:
More remove command functionality

ploppyperson committed:
Format enum when displayed in cmd

ploppyperson committed:
Add chunk option to forcestack

ploppyperson committed:
Fix inconsistency

ploppyperson committed:
Increment version

ploppyperson committed:
Add ability to set a fixed offset for armor stand name tags

ploppyperson committed:
Support 1.19.3

ploppyperson committed:
Fix issues with fake armour stands

ploppyperson committed:
Increment version

Aurelien30000 committed:
Fix PlayerArmorStandListener "leak" (#270)

Fix missed config path

ploppyperson committed:
Make it so jobs rewards are multiplied

ploppyperson committed:
Remove apache commons dependency

ploppyperson committed:
Remove the rest of apache commons

Aurelien30000 committed:
Fix small typo

Aurelien30000 committed:
Remove debug code

ploppyperson committed:
Update for 1.19.4

ploppyperson committed:
Fix ProtocolLib on 1.19x servers

ploppyperson committed:
Cleanup

ploppyperson committed:
Increment version

Yive committed:
Add support for Folia

Yive committed:
Update to latest Folia build

ploppyperson committed:
Fix checking for incorrect NMS version

ploppyperson committed:
Fix for #264

Bees coming from hives trigger creature spawn event with SpawnReason BEEHIVE, however when using Paper's Entity#getEntitySpawnReason they retain their original spawn reason.

We could change it so that the spawn event uses the Paper method for checking blacklists etc, but there would be problems for spigot users, requiring checks for use of Paper. But if an entity was previously destacked - they would become stacked again.

ploppyperson committed:
Maybe fix NPE (#274)

ploppyperson committed:
Update NMS for 1.20

ploppyperson committed:
Increment version

Yomamaeatstoes committed:
Create Schedulers

Yomamaeatstoes committed:
Merge branch 'master' into master-1

ploppyperson committed:
Fix merge error and pom

ploppyperson committed:
Adds folia support

ploppyperson committed:
Remove unneeded alphabet check

ploppyperson committed:
Temp fix for 1.20.2

ploppyperson committed:
Increment version

HSGamer committed:
Merge branch 'master' into folia

# Conflicts:
#	pom.xml

HSGamer committed:
update dependencies

HSGamer committed:
fix the folia check

ploppyperson committed:
Merge branch 'HSGamer-folia' into folia

ploppyperson committed:
Update for 1.20.4

ploppyperson committed:
Remove idea of native version

ploppyperson committed:
Add check for protocollib when getting tag mode value

ploppyperson committed:
Add timeout for check-location

ploppyperson committed:
Increment version

ploppyperson committed:
Update adventure

ploppyperson committed:
Include kyori option library

HSGamer committed:
Merge branch 'master' into folia

# Conflicts:
#	src/main/resources/plugin.yml

HSGamer committed:
fix merge

HSGamer committed:
merge tasks to reduce duplicated code

ploppyperson committed:
Increment version

ploppyperson committed:
Increment version

ploppyperson committed:
Merge branch 'HSGamer-folia' into folia

SuperRonanCraft committed:
moved IS_FOLIA to Utilities

request by `yomamaeatstoes` weird username, but that guy.

ploppyperson committed:
Merge branch 'SuperRonanCraft-folia' into folia

ploppyperson committed:
Replace package checking with string checking for server version

ploppyperson committed:
Remove automatic downloading of StackMobBridge

ploppyperson committed:
Add option to disable knockback for entities

ploppyperson committed:
Update for 1.20.6

ploppyperson committed:
Merge branch '5.9-knockback' into 5.9

ploppyperson committed:
Merge remote-tracking branch 'origin/folia' into 5.9

ploppyperson committed:
Use paper method to get next entity id

ploppyperson committed:
Check when loading entity classes whether entity type is that of a Mob or not

ploppyperson committed:
Make sure to create config for UNKNOWN entity type

ploppyperson committed:
Fix incorrect id used when packets sent after spawn packet is sent

ploppyperson committed:
Increment version

ploppyperson committed:
Actually register listener

ploppyperson committed:
Run on 1.21

ploppyperson committed:
Increment version

ploppyperson committed:
Fix loading on craftbukkit

ploppyperson committed:
Revert to Java 17

ploppyperson committed:
Workaround for #292

In 1.21, fire ticks are applied after death event occurs, so as a temporary? fix need to delay by one tick

ploppyperson committed:
Increment version

ploppyperson committed:
Remove unneeded plugin parameter

ploppyperson committed:
Prevent three-way split when dying with a limit set in config

ploppyperson committed:
Don't take items from inventory when in creative mode

ploppyperson committed:
Initial implementation using display entities

ploppyperson committed:
Remove debug

ploppyperson committed:
Convert to legacy colour codes

ploppyperson committed:
Provide adjusted location as argument

ploppyperson committed:
Fix for #264

Bees coming from hives trigger creature spawn event with SpawnReason BEEHIVE, however when using Paper's Entity#getEntitySpawnReason they retain their original spawn reason.

We could change it so that the spawn event uses the Paper method for checking blacklists etc, but there would be problems for spigot users, requiring checks for use of Paper. But if an entity was previously destacked - they would become stacked again.

(description copied from 50d6ee0)

ploppyperson committed:
Remove adventure library as dependency, use Paper methods

ploppyperson committed:
Fix CME

ploppyperson committed:
Use component name

ploppyperson committed:
Move to TextDisplay for handling name tags

ploppyperson committed:
Remove redundant tag statistics subcommand

ploppyperson committed:
Paper only preparations

ploppyperson committed:
Remove ProtocolLib dependency

ploppyperson committed:
Remove rest of ProtocolLib

ploppyperson committed:
Remove rest of ProtocolLib (2)

ploppyperson committed:
Adjust updater for modrinth

ploppyperson committed:
Version warning

ploppyperson committed:
Merge

ploppyperson committed:
Increment version

adabugra committed:
Fix UnsupportedOperationException

adabugra committed:
Update retired paper repo endpoint

ploppyperson committed:
Update README.md

ploppyperson committed:
Potential fix for #301 - use setPersistent rather than removing on entity remove

ploppyperson committed:
Remove debug

ploppyperson committed:
Make stack data removal method run when entire stack killed (#302)

ploppyperson committed:
Remove tag on removal (#302)

ploppyperson committed:
Remove registration for command argument

ploppyperson committed:
Pom updates

ploppyperson committed:
Increment version

ploppyperson committed:
Add different remove event listener (#302)

ploppyperson committed:
Fix compile (#302)

ploppyperson committed:
Increment version

Excaliburns committed:
Add StackDropItemEvent

Yive committed:
Fix https://github.com/ploppyperson/StackMob-5/issues/309

During the update to using text displays, the check for the tag mode was removed.

ploppyperson committed:
Update for 1.21.8

ploppyperson committed:
Increment version

ploppyperson committed:
Revert "Update for 1.21.8"

This reverts commit 78e31cebb652fd9de69d861a293fa5ec33da49e6.

ploppyperson committed:
Fix NPE (#311)

tgbhy committed:
feat(traits): add Snowman derp trait

Mapacheee committed:
update to 1.21

ploppyperson committed:
Increment version

ploppyperson committed:
Compile for 26.1

ploppyperson committed:
Increment version

ploppyperson committed:
Increment version

ploppyperson committed:
Fix biome specific eggs not being multiplied

ploppyperson committed:
Remove warning on 26.2 (#320)

ploppyperson committed:
Add check of item damage rather than throwing error

ploppyperson committed:
Fix that disabling line-of-sight actually prevents stacking completely (#321)

ploppyperson committed:
Increment version

ploppyperson committed:
Revert paper api version
